flytekit/core/testing.py

```diff
@@ -2,9 +2,9 @@
 from typing import Union
 from unittest.mock import MagicMock
 
-from flytekit.annotated.base_task import PythonTask
-from flytekit.annotated.reference_entity import ReferenceEntity
-from flytekit.annotated.workflow import Workflow
+from flytekit.core.base_task import PythonTask
```

Comment:
i thought you want testing to be a separate module on its own, like `flytekit.testing` ?

Reply:
yeah that's here.

Reply:
so I know it's not a lot of code, but I left the actual implementation in `flytekit.core.testing` and i'm importing them in `flytekit/testing/__init__.py`.  Just for consistency.  I think it's okay?  bit circuitous but that's okay.